### ISSUE TITLE
Counterintuitive `mix ecto.rollback` interface

### ISSUE THREAD

**Issue by staring-frog** [elixir-ecto/ecto_sql#336]
### Current behavior

```
$ mix ecto.rollback --to 20210604145227
12:15:35.434 [info]  == Running 20210617143138 [...] backward
12:15:35.444 [info]  alter table o[...]
12:15:35.471 [info]  == Migrated 20210617143138 in 0.0s
12:15:35.483 [info]  == Running 20210604145227 [...] backward
12:15:35.483 [info]  drop index dial_[...]_index
12:15:35.484 [info]  drop table dial_[...]
12:15:35.486 [info]  == Migrated 20210604145227 in 0.0s
```
Migration specified with `to` is also being rolled back.

### Expected behavior

mix ecto.rollback leaves a repo exactly at `to` migration, i.e. it does "rollback to" specified revision.

That would make more sense, right?
(It would be a one-character fix PR, I guess.)

**Comment by josevalim** [comment]
That's a good point. However, I am worried about changing the current behaviour and breaking the code for anyone who could be relying on this. So my suggestion would be to introduce a new option that matches the current semantics, and then if necessary we could introduce one that mirrors what you propose.

**Comment by staring-frog** [comment]
I don't quite get... Then there would be two options with the same behaviour?.. I thought the usual approach in this case is to add a new option (like `--upto`) with behaviour I've described and the existing one (`--to`) to be marked as deprecated, but still supported for some time.

**Comment by josevalim** [comment]
That’s another option, but we likely shouldn’t call it —up-to as we are going “down”. It could be down-to on rollback and up-to on migrate for consistency.

**Comment by josevalim** [comment]
Ping. Would you like to send a PR, @staring-frog? :)

**Comment by staring-frog** [comment]
I got fired from proSapient (yeah, that place :wink:), hence it became less relevant for me :D It's unlikely that I'll have the time for this issue. But I'll find some time for other two issue I've promised about, after I get back from unplanned vacation :D
Btw "up-to" and "down-to" are hard to remember IMHO: I had hard times recalling each time which way are we going while code diving. Probably [--target, -T] would be easier to remember.
